.changeset/early-trains-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-tech-radar': minor
```

Comment:
Think that this can be a `patch` change as `tech-radar` is <1.0.0

Reply:
Fixed